apps/server/src/schemas/email.schema.ts

```diff
@@ -0,0 +1,19 @@
+import { Prop, Schema, SchemaFactory } from '@nestjs/mongoose';
+import { HydratedDocument } from 'mongoose';
+import { Recipient, RecipientSchema } from './recipient.schema';
+
+export type EmailDocument = HydratedDocument<Email>;
+
+@Schema()
+export class Email {
+  @Prop()
+  subject: string;
```

Comment:
for each `required` field in each schema file, please use the decorator `@prop` with this `required` like below:
` @Prop({ required: true })`

also for fields with string type with some `enum` possible value use this example below:
`@Prop({ enum: ['full_access', 'sending_access'], default: 'sending_access' })
  permission: string; `